runtime/lua/tohtml.lua

```diff
@@ -611,9 +636,7 @@ local function styletable_folds(state)
     end
   end
   if has_folded and type(({ pcall(vim.api.nvim_eval, vim.o.foldtext) })[2]) == 'table' then
-    vim.notify_once(
-      'Info(TOhtml): foldtext returning a table is half supported, HTML may be incorrect'
-    )
+    notify('foldtext returning a table is half supported, HTML may be incorrect')
```

Comment:
What does "half supported" mean? Either it is supported, or not ;) (In other words, this message should be consistent with the others.)

Reply:
notification updated